tests/phpunit/integration/Core/Util/Auto_UpdatesTest.php

```diff
@@ -17,12 +17,13 @@
  * @group Util
  */
 class Auto_UpdatesTest extends TestCase {
+	/**
+	 * Forced auto updates are available for WP >= 5.6 only, so we test
+	 * for a function only available in WordPress 5.6 or greater.
+	 *
+	 * @requires function wp_is_auto_update_forced_for_item
+	 */
 	public function test_sitekit_autoupdate_forced() {
-		// Forced auto updates are available for WP >= 5.6 only.
-		if ( version_compare( get_bloginfo( 'version' ), '5.6', '<' ) ) {
```

Comment:
Just to be clear, this was the only test that explicitly required 5.6 (this condition is missing the "or equals" too mentioned in the comment). Using `@requires function wp_is_auto_update_forced_for_item` is accurate here and also reads clearly as to what the test is about.